public/keymaps/m/marksard_leftover30.json

```
@@ -0,0 +1,35 @@
+{
+    "version": 1,
+    "notes": "",
+    "documentation": "\"This file is a QMK Configurator export. You can import this at <https://config.qmk.fm>. It can also be used directly with QMK's source code.\n\nTo setup your QMK environment check out the tutorial: <https://docs.qmk.fm/#/newbs>\n\nYou can convert this file to a keymap.c using this command: `qmk json2c {keymap}`\n\nYou can compile this keymap using this command: `qmk compile {keymap}`\"\n",
+    "keyboard": "marksard/leftover30",
+    "keymap": "default",
```

This will help me know if this file needs to be updated if you change the keymap on the qmk_firmware repository.

```suggestion
    "commit": "6a79d99ea2d4c72200953b35d51bb9b9d784a54b",
    "keymap": "default",
```

---

I understand. I'm working on the qmk firmware a little bit now, so I'll fix this one when it's done.

---

I added a commit id.
Thank you for your support.